app/assets/stylesheets/styles.scss

```diff
@@ -2,4 +2,65 @@
 
 .modal-dialog {
   max-width: 60%;
-}
\ No newline at end of file
+}
+
+.content {
+  max-width: 60%;
+  flex-grow: 1;
+}
+
+.navbar {
+  flex-direction: row;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-link {
+  color: white;
+  text-decoration: none;
+}
+
+.nav-pad {
+  padding-right: 0.5rem;
+  padding-left: 0.5rem;
+}
+
+.nav-pad :hover {
+  cursor: pointer;
+}
+
+.align-center {
+  display: flex;
+  align-items: center;
+}
+
+.sticky-header {
+  position: sticky;
+  top: 0;
+}
+
+.scale-icon-06 {
+  transform: scale(0.6);
+}
+
+.scale-icon-08 {
+  transform: scale(0.8);
+}
+
+.border-r-l {
+  border-right: 1px solid;
+  border-left: 1px solid;
+}
+
+.top-bar-color {
+  background-color: #3b7bbe
```

Comment:
Aren't there variables available from the puzzle shell for this?
If not define the color globally as a variable so we can use it in multiple places. Same goes for the grey below.